src/Appwrite/Utopia/Response/Model/Key.php

```diff
@@ -27,6 +27,12 @@ public function __construct()
                 'default' => '',
                 'example' => 'My API Key',
             ])
+            ->addRule('expire', [
+                'type' => self::TYPE_INTEGER,
+                'description' => 'Key expiration timestamp.',
```

Comment:
```suggestion
                'description' => 'Key expiration in Unix timestamp.',
```

Again, very important to be very accurate in the descriptions. These are later used in the docs.